htdocs/takepos/takepos.php

```diff
@@ -446,11 +446,11 @@ function MoreActions(totalactions){
 $menus[$r++]=array('title'=>$langs->trans("Customer"),
 					'action'=>'Customer();');
 $menus[$r++]=array('title'=>$langs->trans("BackOffice"),
-                   'action'=>'window.open(\''.DOL_URL_ROOT.'\', \'backoffice\');');
+					'action'=>'window.open(\''.(empty(DOL_URL_ROOT)?'/':DOL_URL_ROOT).'\', \'_self\');');
 $menus[$r++]=array('title'=>$langs->trans("ValidateBill"),
 					'action'=>'CloseBill();');
 $menus[$r++]=array('title'=>$langs->trans("Logout"),
-                   'action'=>'window.location.href=\''.DOL_URL_ROOT.'/user/logout.php\';');
+					'action'=>'window.location.href=\''.(empty(DOL_URL_ROOT)?'/':DOL_URL_ROOT).'user/logout.php\';');
```

Comment:
If DOL_URL_ROOT is not empty it is something like
/dolibarr_dev/htdocs
So here the href will be wrong and will be
/dolibarr_dev/htdocsuser/logout.php  (the / is missing after htdocs)

Reply:
@eldy yes it's fixed, thanks